Gemfile

```diff
@@ -9,3 +9,5 @@ group :test do
   gem 'shoulda', require: false
   gem 'rubocop', require: false
 end
+
+gem "codeclimate-test-reporter", group: :test, require: nil
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)